layouts/partials/head.html

```diff
@@ -7,15 +7,16 @@
 <title>{{ .Site.Title }}</title>
 {{ "<!-- Stylesheets -->" | safeHTML }}
 <link href="{{ .Site.BaseURL }}css/bootstrap.min.css" rel="stylesheet">
-<link href="{{ .Site.BaseURL }}css/agency.css" rel="stylesheet">
+<link href="{{ .Site.BaseURL }}css/ictgj-2017-feb.css" rel="stylesheet">
 {{ range .Site.Params.custom_css }}
     <link rel="stylesheet" href="{{ $.Site.BaseURL }}{{ . }}">
 {{ end }}
 
 {{ "<!-- Custom Fonts -->" | safeHTML }}
 <link href="{{ .Site.BaseURL }}font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
 <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
-<link href="https://fonts.googleapis.com/css?family=Kaushan+Script" rel="stylesheet" type="text/css"><link href="https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic" rel="stylesheet" type="text/css">
```

Comment:
Why remove?  I think this was added as a suggestion so a similar-looking font would load for linux users or something?  Don't quite remember.  Some browser on some OS didn't look great with plain sans-serif or something.